docs/guidance/rayservice.md

```diff
@@ -1,103 +1,115 @@
-## Ray Services (alpha)
+# Ray Services (alpha)
 
 > Note: This is the alpha version of Ray Services. There will be ongoing improvements for Ray Services in the future releases.
 
-### Prerequisites
+# Prerequisites
 
-* Ray 2.0 or newer.
+This guide focuses solely on the Ray Serve multi-application API, which is available starting from Ray version 2.4.0.
+
+* Ray 2.4.0 or newer.
 * KubeRay 0.6.0 or newer.
```

Comment:
On this line, let's specify that the nightly will work too.

Reply:
Updated https://github.com/ray-project/kuberay/pull/1235/commits/60ac0ef09b05b3d768939492a6d6907d6a3cbb1f